multiple_listeners/config_for_tests.py

```diff
@@ -0,0 +1,1683 @@
+######################################
+# This file has been auto generated. #
+######################################
+backends = [
+
+    {
+        'id': 'nginx',
+        'type': 'nginx',
+        'port': '8000',
+        'status_uri': 'http://${server_ip}:8000/nginx_status',
```

Comment:
For https://github.com/tempesta-tech/tempesta/issues/689 we also need IPv6 backend (Nginx) addresses - this can be done in a separate test or all the tests can use IPv6 server addresses (we already have a lot of tests with Nginx listening on IPv4).